compiler/src/iree/compiler/InputConversion/StableHLO/Passes.td

```diff
@@ -18,6 +18,12 @@ def ConvertStableHloToLinalg :
                         "transpose) when possible, instead of linalg.generic">];
 }
 
+def ConvertStableHloToLinalgExt :
+    Pass<"iree-stablehlo-to-linalg-ext", "func::FuncOp"> {
+  let summary =
+      "Convert from XLA-HLO ops to LinalgExt ops and distribute to Flow ops";
+}
```

Comment:
Update summary, at least "XLA-HLO" to "StableHLO"

Reply:
Thanks!